website/docs/patterns/data-visualization/color-usage/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Color usage
```

Comment:
Because of the nested organization this results in the title of the card on the patterns page solely being "Color usage", rather than being directly tied to data visualization, which I think is pretty confusing. A couple of different solutions to this (understanding that we discussed the hierarchy in the google doc):
- Since there isn't anything else nested inside of the Data Visualization category at this time, elevate this up to Data Visualization Color Usage (kind of long)
- Change to "Visualization Color Usage" or something like that
- Remove the "Color Usage" section and instead turn everything in the tab underneath a Data Visualization page (essentially replace "Guidelines" with "Color Usage")

Are there any other solutions that folks can think of? Or is this non-issue?

Reply:
I think maybe your last suggestion makes sense so that way we won't need multiple cards for different aspects of data visualization.

Reply:
Made the update and pushed up. Thanks for that suggestion I think it works well and it will be easy to add new tabs if/when needed.